src/coreclr/jit/llvm.cpp

```diff
@@ -125,6 +129,16 @@ Value* Llvm::getGenTreeValue(GenTree* op)
     return _sdsuMap->at(op);
 }
 
+TypeDescriptor Llvm::getTypeDescriptor(CORINFO_CLASS_HANDLE structHandle)
+{
+    if (_typeDescriptorsMap->find(structHandle) == _typeDescriptorsMap->end())
+    {
+        _typeDescriptorsMap->insert({structHandle, _getTypeDescriptor(_thisPtr, structHandle)});
+    }
+
+    return _typeDescriptorsMap->at(structHandle);
+}
```

Comment:
Appears unused (as well as the associated map).

Reply:
Thanks, yes you are right..